exercises/aux_files.md

```diff
@@ -0,0 +1,133 @@
+# Exercises: Using aux files
+
+We want to explore auxiliary files in these exercises, which allow you to load up contextual information
+alongside your data. 
+
+## 1. Aux files with a plain text file
+
+In this example we will add a simple linear trend to ten years of data, and view the results using
+`ncview`, to confirm that everything works fine. A practical use case here would be to add a varying
```

Comment:
`ncview` won't work with fesom data, right? If we want to visualize it a regular-grid data set should be prepared, for example based on echam output and visualize it with ncview

Reply:
No, you need something `ncview`-friendly. I would suggest we go with fake data here on a 360x180 grid.